Template/Tag/CustomHtmlTag.web.js

```diff
@@ -125,6 +125,12 @@
                 newScript.setAttribute('type', element.getAttribute('type'));
             }
 
+            // If this script is executing using a nonce, set the nonce on its children too.
+            var scriptWithNonce = document.querySelector('script[nonce]');
```

Comment:
here we would typically use `TagManager.dom.bySelector` if that works?

Reply:
@tsteur Yes. It looks like that works. I just have to add a little more logic because it returns an array instead of the first element.

Reply:
👍 great. Just fyi we're using these kind of methods for better browser compatibility so that no logic breaks in case the method isn't defined in an older browser.